fuzzers/test

```diff
@@ -0,0 +1 @@
+hello
```

Comment:
need to remember to remove this. just a test i think :)

Reply:
Yeah I did.